.github/workflows/continuous-e2e-single-vfn-test.yaml

```diff
@@ -19,5 +21,4 @@ jobs:
       FORGE_RUNNER_DURATION_SECS: 480
       FORGE_RUNNER_TPS_THRESHOLD: 5000
       FORGE_TEST_SUITE: single_vfn_perf
-      POST_TO_SLACK: true
```

Comment:
why not post to slack :^)

Reply:
Good catch. I think I tried to see if the previous permissions issue was due to slack :P